ee/ucp/kubernetes/install-cni-plugin.md

```diff
@@ -1,98 +1,117 @@
+keywords: ucp, Kubernetes, cni, Container Networking Interface, flannel, weave, calico
 ---
 
-For Docker Universal Control Plane, [Project Calico](https://docs.projectcalico.org/v3.0/introduction/) 
-provides the secure networking functionality for the container communication with Kubernetes.
+For Docker Universal Control Plane, [Project
+Calico](https://docs.projectcalico.org/v3.7/introduction/) provides the secure
+networking functionality for the container to container communication within
+Kubernetes. The Universal Control Plane handles the lifecycle of Project Calico,
+installing and upgrading the project as part of UCP. Additionally, the Project
+Calico deployment installed is fully supported, with Docker Support providing
+guidance on the [CNI components](https://github.com/projectcalico/cni-plugin).
 
-Docker EE supports Calico and installs the 
-built-in [Calico](https://github.com/projectcalico/cni-plugin) plugin, but you can override that and 
-install a Docker certified plugin.
+At install time, the Universal Control Plane can be configured to not install
+the Calico CNI Plugin, allowing Platform Operators to install alternative CNI
+plugins post installation to support alternative use cases. In this model all
+Universal Control Plane components are still fully supported by Docker Support,
+however for guidance and support on the 3rd party CNI components, the platform
+operator will need to contact that 3rd party.
 
-> **Note**: The `--cni-installer-url` option is deprecated as of UCP 3.1. It is replaced by the `--unmanaged-cni` option.
+## Installing a Unmanaged CNI Plugin on Docker UCP
 
-# Install UCP with a custom CNI plugin
+Once a platform operator has followed the [System
+Requirements](/ee/ucp/admin/install/system-requirements/) documentation, and
+taken into consideration any requirements for the custom CNI plugin. The Docker
+UCP installation command can be ran with a `--unmanaged-cni` flag to bring up
+the platform.
 
-Modify the [UCP install command-line](../admin/install/index.md#step-4-install-ucp)
-to add the `--cni-installer-url` [option](/reference/ucp/3.0/cli/install.md),
-providing a URL for the location of the CNI plugin's YAML file:
+This command will install the Universal Control Plane, and bring up components
+like the UCP UI and the RBAC engine. Universal Control Planes components that
+require Kubernetes Networking, such as Metrics, will not start and will stay in
+a `Container Creating` state in Kubernetes, until a CNI is installed. 
+
+### Install UCP without a CNI Plugin
+
+Once connected to a manager node, with the Docker Enterprise Engine installed, you are ready to bring up UCP with the `--unmanaged-cni` flag.
 
 ```bash
 docker container run --rm -it --name ucp \
   -v /var/run/docker.sock:/var/run/docker.sock \
   {{ page.ucp_org }}/{{ page.ucp_repo }}:{{ page.ucp_version }} install \
   --host-address <node-ip-address> \
-   --unmanaged-cni <true|false> \
+  --unmanaged-cni true \
```

Comment:
Do we need to specify 'true'?  My experience is when I add true, the install doesn't prompt for Admin username or passwd.

Reply:
Updated in latest commit. Thanks @uday-shetty